backend/qemu.pm

```diff
@@ -201,6 +201,10 @@ sub can_handle {
 
         return if $vars->{QEMU_DISABLE_SNAPSHOTS};
 
+        for my $i (1 .. $vars->{NUMDISKS}) {
+            return if $vars->{"HDDMODEL_$i"} eq 'nvme';
```

Comment:
From your log, I could see uninitialized value warning here, guess if not define all HDDMODEL_$i in settings, better to add `defined` statement to avoid this warning.